### PR TITLE
Editing style for User View 

### PR DESCRIPTION
**Pull request by s-luong07**
## Editing style for User View 

**Description:** 
- fixed scaling problem of the header and added **search icon** to **search rooms** ( need to ad function and retouch so it can pop up)
- edited buttons in AllRoomsBody, replaced **like icon** by **arrow icon** 
- edited Create Post Form
-  replace replaced **like icon** by **arrow icon**  in Post Card 
Screenshot (992)
Screenshot (997)
Screenshot (999)
Screenshot (1001)

### CODE REVIEW

File: src/components/post/Post.js [hunk 1]
```diff
@@ -198,12 +198,12 @@ const Post = (props) => {
                   <p>
                     {liked ? (
                       <i
-                        className="fas fa-thumbs-up"
+                        class="fas fa-chevron-up"
```
Comment (jakegrella) [inline src/components/post/Post.js]: shouldn't these stay as className since it's a react component?

Comment (s-luong07) [reply src/components/post/Post.js]: if I kept className then it wouldn't show the up arrow

Comment (s-luong07) [reply src/components/post/Post.js]: so after I check it again .I  can change it to classsName, honestly don't know why first time I tried it didn't work, you're right

**Review by jamesjlundin — APPROVED** [review]

**Review by jakegrella — APPROVED** [review]
looks good now!

**Pull request merged**